developer/web-services/index.md

```diff
@@ -10,26 +10,45 @@ categories:
   - Developer
 date: 2011-06-30 18:23:51 -0500
 ---
-<i class="fas fa-2x fa-fw fa-magic"></i> The [AGRC's Web API](http://api.mapserv.utah.gov) allows for developers to work with the data and services that AGRC maintains and creates. You can query any data layer in our SGID or you can geocode addresses against the most accurate statewide roads data set to name a few highly popular uses. All of our [widgets and add-ins]({{ "/developer/widgets/" | prepend: site.baseurl }}) make full use of our web API's.
+<i class="fas fa-2x fa-fw fa-magic"></i> The Automated Geographic Reference Center (AGRC) is the State of Utah's map technology coordination office. The AGRC creates, maintains, and stores geospatial data in the State Geographic Information Database (SGID), a one-stop shop to hundreds of data layers developed, aggregated, or acquired by state government. [AGRC's Web API](https://api.mapserv.utah.gov) is an http-enabled service for accessing this valuable geospatial data. 
```

Comment:
Is it necessary to describe what the agrc is here?

Reply:
I think I added in that text to give the API info a little more context and introduction to the page. But whether to include the info, in my opinion, is mostly dependent on our audience for this particular page. For someone who's a little less familiar with AGRC and what the office does, the extra couple lines may aid in clarity. For instance, when I first read this text, I don't think I knew very much about the SGID, so for me, it would've been helpful to know that the AGRC maintains all the data in the SGID (and to spell out what the SGID is), and then APIs allow people to work with that data. But for someone more familiar with AGRC, APIs, geocoding, etc., maybe that info is unnecessary. Totally your call :)